kong/plugins/jwt/handler.lua

```diff
@@ -41,20 +41,28 @@ local function retrieve_token(conf)
     end
   end
 
-  local authorization_header = kong.request.get_header("authorization")
-  if authorization_header then
-    local iterator, iter_err = re_gmatch(authorization_header, "\\s*[Bb]earer\\s+(.+)")
-    if not iterator then
-      return nil, iter_err
-    end
+  local request_headers = kong.request.get_headers()
+  for _, v in ipairs(conf.header_names) do
+    local token_header = request_headers[v]
+    if token_header then
+      if type(token_header) == "table" then
+        token_header = token_header[1]
+      end
+      local iterator, iter_err = re_gmatch(token_header, "\\s*[Bb]earer\\s+(.+)")
+      if not iterator then
+        ngx.log(ngx.ERR, "[jwt]: ", iter_err)
+        break
+      end
 
-    local m, err = iterator()
-    if err then
-      return nil, err
-    end
+      local m, err = iterator()
+      if err then
+        ngx.log(ngx.ERR, "[jwt]: ", err)
```

Comment:
needs to be replaced with `kong.log.err`

Reply:
Hi guys
Who can fix this PR?
Thank you in advance